Hummingbird.docc/HummingbirdAuth/AuthenticatorMiddleware.md

```diff
@@ -63,7 +63,7 @@ router.middlewares.add(BasicAuthenticator())
 router.middlewares.add(IsAuthenticatedMiddleware<User>())
 ```
 
-Or you can use ``HummingbirdAuth/LoginCache/require(_:)`` to access the authentication data. In both of these cases if data is not available a unauthorised error is thrown and a 404 response is returned by the server.
+Or you can use ``AuthRequestContext/identity-swift.property`` to access the authenticated identity.
```

Comment:
HummingbirdAuth/AuthRequestContext?

How does the `identity-swift.property` work? I've not seen that before. `identity` should be enough.

Reply:
That's what Xcode completes it with